tests/test_ddpo_trainer.py

```diff
@@ -0,0 +1,77 @@
+import gc
```

Comment:
(nit) can you add the copy right here with your GH handle + Hugging Face team: https://github.com/lvwerra/trl/blob/main/trl/import_utils.py#L1